.github/workflows/rzk.yml

```diff
@@ -17,9 +21,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check all files
+      - name: Typecheck Rzk files
         uses: rzk-lang/rzk-action@v1
         with:
           rzk-version: latest
           # rzk-version: v0.4.1.1   # example of a specific version
-          files: src/hott/* src/simplicial-hott/*
+          files: ''
```

Comment:
I think with the release of rzk-action@v1.1.0 we can drop the `files` input:
```suggestion
```

We could also drop `rzk-version`, but I think it is nice to have it explicitly mention that latest `rzk` is used.

Reply:
Does the new default already apply to version 1 of this action?

Reply:
I assumed it would, but apparently it does not work like that.

Reply:
I think this is because I had an explicit `v1` tag. I've renamed it to `v1.0.0` just now.

Reply:
No, apparently, I have to maintain `v1` separately to point to the latest `v1.*.*` tag.

Reply:
Oh, I see, I need a `v1` branch.

Reply:
Now `v1` works properly.

Reply:
Wait, do you actually want to retroactively change that?

Reply:
The intent for `v1` was always to point to the latest `v1.*.*`, it's the same with other GitHub Actions, I just did not properly set it up previously.

Reply:
Ah gotcha, that makes sense